server/services/allocationsService.ts

```diff
@@ -18,6 +18,10 @@ export default class AllocationsService {
     this.config = config
   }
 
+  async getLaoStatus(crn: string, token: string): Promise<boolean> {
+    return (await this.restClient(token).get({ path: `/cases/unallocated/${crn}/cases/restricted` })) as boolean
```

Comment:
This endpoint should be **/cases/unallocated/{crn}/restricted**

Reply:
fixed